ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureCloudDestination.java

```diff
@@ -64,40 +67,37 @@
   private final DocumentClient documentClient;
   private final String cosmosCollectionLink; // eg "/dbs/ambry-metadata/colls/blob-metadata"
```

Comment:
could you move the comment to a separate line? (make the code style more consistent)

Reply:
removed comment